index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Portfolio</title>
+  <header>
+      <img src="img/My Logo.svg" alt="humburger" class="Logo">
+      <img src="img/Union.svg" alt="humburger"  class="Union" >
+  </header>
+</head><br><br><br><br>
+<body>
+  <div id="my-section">
+    <section class="container">
+      <h1>I'm Edgah.<br>
+        Glad to see<br>
+        you!</h1>
+      <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+       <img src="img/LET’S CONNECT.svg" alt="humburger"><br><br>
+     </section>
+     <section>
+     <ul class="social-icons">
+      <li>
+          <img src="img/twitter icon.svg">
+      </li>
+      <li>
+          <img src="img/Linkedin icon.svg">
+      </li>
+      <li>
+          <img src="img/Mobile icon.svg">
+      </li>
+      <li>
+        <img src="img/git icon.svg">
+      </li>
+      <li>
+        <img src="img/peace icon.svg">
+      </li>
+      </ul>
+    </section><br><br><br><br><br><br><br><br>
+  </div>
+ <div>
+   <section class="together">
+      <img src="img/Snapshoot Portfolio.svg">
+      <div class="textp">
+      <h2>Tonic</h2>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph" id="devend">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph" id="unique">2023</p>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <button type="button" class="my-button" >HTML</button>
+      <button type="button" class="my-button" >CSS</button>
+      <button type="button" class="this-button" >Javascript</button><br><br>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+    <section class="multipost">
+      <img src="img/multipost.svg" class="postsvg">
+      <h2>Multi-Post<br>
+        Stories</h2><br>
+        <p class="inline-paragraph">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph" >Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph" >2023</p>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <button type="button" class="my-button" >HTML</button>
+        <button type="button" class="my-button" >CSS</button>
+        <button type="button" class="this-button" >Javascript</button><br><br>
```

Comment:
Great job so far. Please use the `ul` and `li` tags to list this button since you need to follow the project requirement.
![Screenshot from 2023-06-03 19-48-06](https://github.com/Edgahkipkemoi/mobile/assets/101523582/08fd506f-a1a9-4452-9be0-d127d1309fdf)